### DISCUSSION TITLE
Adding a visual novel command

### DISCUSSION THREAD

**Discussion by it0na72** [mistval/kotoba#163] [Ideas]
Hello. I was just wondering if it was possible to add a similar command like "k!manga gintama" or "k!anime ginga eiyuu densetsu" for visual novels instead (something like "k!vn saya no uta" and it would display the synopsis with the title just like this: https://i.imgur.com/yzdZswT.png)

VNDB has the API available and it would be amazing to search for visual novels in Discord. 

Thanks in advance.

**Comment by mistval** [comment]
Hi, that looks feasible, and sounds like it would be fun. I will try to do this when I can make the time.
